tests/hardhat/Fork/liquidation.ts

```diff
@@ -0,0 +1,311 @@
+        await expect(
+          vUSDT.connect(acc1Signer).liquidateBorrow(acc2, usdtBorrowAmount, vUSDT.address),
+        ).to.be.revertedWithCustomError(comptroller, "MinimalCollateralViolated");
+      });
+
+      it("Should revert when liquidation is called through vToken and no shortfall", async function () {
+        // Mint and Increase collateral of the user
+        const udnerlyingMintAmount = convertToUnit("1", 30);
+        await usdd.connect(acc2Signer).faucet(udnerlyingMintAmount);
+        await usdd.connect(acc2Signer).approve(vUSDD.address, udnerlyingMintAmount);
+
+        await vUSDD.connect(acc2Signer).mint(udnerlyingMintAmount);
+
+        // Liquidation
+        await expect(
+          vUSDT.connect(acc1Signer).liquidateBorrow(acc2, usdtBorrowAmount, vUSDT.address),
+        ).to.be.revertedWithCustomError(comptroller, "InsufficientShortfall");
+      });
+
+      it("Should revert when liquidation is called through vToken and trying to seize more tokens", async function () {
+        await comptroller.setMinLiquidatableCollateral(0);
+        // Mint and Increase collateral of the user
+        await priceOracle.setDirectPrice(usdd.address, convertToUnit("1", 5));
+        // Liquidation
+        await expect(vUSDT.connect(acc1Signer).liquidateBorrow(acc2, 201, vUSDD.address)).to.be.revertedWith(
+          "LIQUIDATE_SEIZE_TOO_MUCH",
+        );
+      });
+
+      it("Should revert when liquidation is called through vToken and trying to pay too much", async function () {
+        // Mint and Incrrease collateral of the user
+        await comptroller.setMinLiquidatableCollateral(0);
+        const udnerlyingMintAmount = convertToUnit("1", 18);
+        await usdd.connect(acc2Signer).faucet(udnerlyingMintAmount);
+        await usdd.connect(acc2Signer).approve(vUSDD.address, udnerlyingMintAmount);
+
+        await expect(vUSDD.connect(acc2Signer).mint(udnerlyingMintAmount)).to.emit(vUSDD, "Mint");
+        // price manipulation and borrow to overcome insufficient shortfall
+
+        await priceOracle.setDirectPrice(usdd.address, convertToUnit("1", 5));
+        // Liquidation
+        await expect(
+          vUSDT.connect(acc1Signer).liquidateBorrow(acc2, convertToUnit("1", 18), vUSDD.address),
+        ).to.be.revertedWithCustomError(comptroller, "TooMuchRepay");
+      });
+
+      it("liquidate user", async () => {
+        await comptroller.setMinLiquidatableCollateral(0);
+        await priceOracle.setDirectPrice(usdd.address, convertToUnit("100", 15));
+        const borrowBalance = await vUSDT.borrowBalanceStored(acc2);
+        const closeFactor = await comptroller.closeFactorMantissa();
+        const maxClose = (borrowBalance * closeFactor) / 1e18;
+        const result = vUSDT.connect(acc1Signer).liquidateBorrow(acc2, maxClose.toString(), vUSDD.address);
+        await expect(result).to.emit(vUSDT, "LiquidateBorrow");
```

Comment:
I would check more values:
* new borrow balance of acc2 // it should be lower
* new vUSDT balance of acc2 // it should be lower
* new vUSDT balance of acc1Signer // it should be higher
* new reserves in the vUSDT market // it should be higher

I would expect exact numbers, not only lower or greater checks